pkgs/development/tools/build-managers/buildbot/plugins.nix

```diff
@@ -0,0 +1,90 @@
+{ stdenv,
+  fetchurl,
+  pythonPackages
+}:
+
+let
+  buildbot-pkg = pythonPackages.buildPythonPackage rec {
+    name = "buildbot-pkg-0.9.0rc1";
+    version = "0.9.0rc1";
+
+    src = fetchurl {
+      url = "https://pypi.python.org/packages/8f/37/6de8734299b94c6d853d2380f0a2cdadf00d24bbac6b8947ea2d3848d15d/buildbot-pkg-0.9.0rc1.tar.gz";
+      sha256 = "0bbbf10361087493cc6ee62e077863eafe1b3e24c43a8429a6eb1a75b35359b6";
+    };
+
+    propagatedBuildInputs = with pythonPackages; [ setuptools ];
+
+    # doesn't seem to break without this...
+    patchPhase = ''
+      sed -i.bak -e '/"setuptools >= 21.2.1",/d' setup.py
+    '';
+
+    meta = with stdenv.lib; {
+      homepage = http://buildbot.net/;
+      description = "Buildbot Packaging Helper";
+      maintainers = with maintainers; [ nand0p ryansydnor ];
+      platforms = platforms.all;
+    };
+  };
+
+in {
+
+  www = pythonPackages.buildPythonPackage rec {
+    name = "buildbot-www-0.9.0rc1";
+    version = "0.9.0rc1";
+    format = "wheel";
```

Comment:
you've explained in the thread why you use a wheel, but please add it here as a comment.